src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/VariantsToTable.java

```diff
@@ -28,61 +28,77 @@
- * downstream analyses. It can extract both INFO/site-level fields and FORMAT/sample-level fields.</p>
+ *     This tool extracts specified fields for each variant in a VCF file to a tab-delimited table, which may be easier
+ *     to work with than a VCF. By default, the tool only extracts PASS or . (unfiltered) variants in the VCF file. Filtered variants may be
+ *     included in the output by adding the --show-filtered flag. The tool can extract both INFO (i.e. site-level) fields and
+ *     FORMAT (i.e. sample-level) fields.
+ * </p>
  *
  * <h4>INFO/site-level fields</h4>
- * <p>Use the `-F` argument to extract INFO/site-level fields, which will appear as a single column each in the output
- * file.  The field can be any standard VCF column (CHROM, ID, QUAL) or any annotation name in the INFO field (AC=10).
- * In addition, there are specially supported values like EVENTLENGTH (length of the event), TRANSITION (for SNPs), HET
- * (count of het genotypes), HOM-REF (count of homozygous reference genotypes), HOM-VAR (count of homozygous variant
- * genotypes), NO-CALL (count of no-call genotypes), TYPE (the type of event), VAR (count of non-reference genotypes),
- * NSAMPLES (number of samples), NCALLED (number of called samples), GQ (from the genotype field; works only for a file
- * with a single sample), and MULTI-ALLELIC (is the record from a multi-allelic site).</p>
+ * <p>
+ *     Use the `-F` argument to extract INFO fields; each field will occupy a single column in the output
+ * file.  The field can be any standard VCF column (e.g. CHROM, ID, QUAL) or any annotation name in the INFO field (e.g. AC, AF).
+ * The tool also supports the following additional fields:
+ * <ul>
+ *     <li> EVENTLENGTH (length of the event) </li>
+ *     <li> TRANSITION (1 for a bi-allelic transition (SNP), 0 for bi-allelic transversion (SNP), -1 for INDELs and multi-allelics) </li>
+ *     <li> HET (count of het genotypes) </li>
+ *     <li> HOM-REF (count of homozygous reference genotypes) </li>
+ *     <li> HOM-VAR (count of homozygous variant genotypes) </li>
+ *     <li> NO-CALL (count of no-call genotypes) </li>
+ *     <li> TYPE (type of variant, possible values are NO_VARIATION, SNP, MNP, INDEL, SYMBOLIC, and MIXED </li>
+ *     <li> VAR (count of non-reference genotypes) </li>
+ *     <li> NSAMPLES (number of samples) </li>
+ *     <li> NCALLED (number of called samples) </li>
+ *     <li> MULTI-ALLELIC (is this variant multi-allelic? true/false) </li>
+ * </ul>
  *
  * <h4>FORMAT/sample-level fields</h4>
- * <p>Use the `-GF` argument to extract FORMAT/sample-level fields, which will appear as a single column each per
- * sample in the output file.</p>
+ * <p>
+ *     Use the `-GF` argument to extract FORMAT/sample-level fields. The tool will create a new column per sample
+ *     with the name "SAMPLE_NAME.FORMAT_FIELD_NAME" e.g. NA12878.GQ, NA12877.GQ.
```

Comment:
It says below that the sample names will be alphabetized, in which case I'd put 77 before 78 to avoid confusion.

Reply:
good catch, done